### DISCUSSION TITLE
`handlr-regex` AUR packages up for adoption

### DISCUSSION THREAD

**Discussion by Anomalocaridid** [Anomalocaridid/handlr-regex#40] [Announcements]
As of my writing this, I have officially disowned the `handlr-regex` and `handlr-regex-bin` AUR packages and they are now orphaned packages.

Just in case it is not clear, they are still there to install, but will not be updated until someone else decides to adopt This also should not affect any packages in non-Arch-based Linux distros, although I only maintain the one in nixpkgs.

I encourage anyone that relies on either of these packages to consider adopting one or both of these packages.

A few months ago, I moved from using an Arch-based Linux distro to NixOS. As such, it's significantly more inconvenient to maintain the `handlr-regex` and `handlr-regex-bin` AUR packages because I do not have access to the Arch-specific tooling it requires. I could create an Arch VM on my system and use that, but I would rather not have to do that just for two packages on a Linux distro I do not intend to return to, even if I only infrequently make changes like I have been.

In hindsight, I really should have made an announcement (and maybe a `handlr-regex-git` package) like this when I moved to NixOS rather than right after a new release. However, I forgot that maintaining AUR packages requires Arch-specific tools and did not think to check that they were available in nixpkgs. I thought I could just get away with updating versions and hashes until someone complains about something breaking, but even for that, I need `updpkgsums`.

I sincerely apologize for any trouble this causes for anyone.

Thank you for your time.

Update (12/7/2023):

As of a few days ago, the `handlr-regex` AUR package has been adopted!

**Comment by gnarlsnishi** [comment]
Thank you for your work on this project! Unfortunately I do not have the know-how to take up maintenance of the AUR packages, but I hope someone else is able to!

**Comment by Anomalocaridid** [comment]
A while ago, the `handlr-regex` AUR package became an official package (replacing the `handlr` package, even), so there is no further need to worry about this.
